dbfit-java/core/src/main/java/dbfit/util/DbParameterAccessor.java

```diff
@@ -40,17 +42,18 @@ public DbParameterAccessor clone() {
     }
 
     @SuppressWarnings("unchecked")
-    public DbParameterAccessor(String name, Direction direction, int sqlType, Class javaType, int position) {
-        this(name, direction, sqlType, null, javaType, position);
+    public DbParameterAccessor(String name, Direction direction, int sqlType, Class javaType, int position, Map<Class<?>, TypeTransformer> typeSpecifers) {
```

Comment:
Maybe something in the shape of `TypeTransformerFactory` is good enough (or we can directly reuse it and just create a separate instance ... dbfitToJDBCTransformer or something).